cloud/user/user_test.go

```diff
@@ -0,0 +1,42 @@
+package user
+
+import (
+	"bytes"
+	"encoding/json"
+	"io"
+	"net/http"
+	"testing"
+
+	"github.com/astronomer/astro-cli/astro-client"
+	testUtil "github.com/astronomer/astro-cli/pkg/testing"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestUserInvite(t *testing.T) {
```

Comment:
nitpicking: generally the name of the unit test method is `Test` followed by the name of the method for which we are writing the unit test cases. In this case, it would be `TestCreateInvite`.